tests/airgapped/airgap.sh

```diff
@@ -79,6 +79,31 @@ function setup_microk8s() {
   airgap_wait_for_pods "$NAME"
 }
 
+function create_ingress_object_in_kubeflow_namespace() {
+  local NAME=$1
+  if ! lxc exec "$NAME" -- bash -c 'cat <<EOF | microk8s kubectl apply -n kubeflow -f -
```

Comment:
Are all our scripts for airgap assuming we are using microk8s?

Reply:
yes, in the [main script](https://github.com/canonical/bundle-kubeflow/blob/main/tests/airgapped/airgap.sh#L136-L137)